lib/facter/postgres_version.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Facter.add('postgres_version') do
+  confine { Facter::Core::Execution.which('postgres') }
```

Comment:
At least on Debian family, `postgres` is not in `PATH`, so this wouldn't work. Maybe it'd be better to get the version from `psql`, which is in the `PATH`?

Reply:
The problem with that is that `psql`might be a different version, and I would like to ensure that it's actually the database server version that gets reported. I guess this needs a bit more thought.

Reply:
As stated below, perhaps `pg_config --version` could be used.